simulation/manage.py

```diff
@@ -47,15 +47,23 @@ def get_ico_instance(address, compiled_source):
     return ico_instance
 
 
-def add_address_to_whitelist(address, controller_instance):
-    tx_hash = controller_instance.transact(
-        {'from': owner_account}
-    ).addBuyers([address])
-    wait_for_tx(tx_hash, w3, wait_message="Wait for account to be added to whitelist")
-    print("\n\n{} successfully added to whitelist".format(address))
+def add_addresses_to_whitelist(address, file_path, controller_instance):
```

Comment:
Not critical in this case (the usage of the function is very limited), but in general, the function interface is not clear, it is not possible to say which parameters are optional and which are required looking at the function definition.

Especially because we now have complex condition for parameters: one of `address` or `file_path` is required and the other one should be `None` (or will be ignored).

It would be better to leave two functions: `add_addresses_to_whiltelist(addresses, controller)` and `add_addresses_to_whitelist_from_csv(file_path, controller)` (this one would parse csv and call the `add_addresses_to_whitelist).